exir/delegate.py

```diff
@@ -151,7 +153,7 @@ def is_lowered_module(obj: Any) -> bool:
     def get_lowered_module_name(
         root: torch.nn.Module,
         # pyre-ignore: Undefined or invalid type [11]: Annotation `LoweredBackendModule` is not defined as a type.
-        lowered_module: LOWERED_BACKEND_MODULE_TYPE,  # noqa
```

Comment:
Is this correct?

Reply:
Yeah, it was a bug before. 

LOWERED_BACKEND_MODULE_TYPE was not a python type, but just a string

https://github.com/pytorch/executorch/blob/2dedc9e0e39047269b7762652b593c5c53883168/exir/delegate.py#L42

Reply:
My guess for the pyre-ignores is likely due to pyre can't recognize string types. However we have to do that because delegate.py and lowered_backend_module.py is cross dependency (not really cross dependency, but delegate.py needs the forward declaration from lowered_backend_module.py. This is supported in c++, but not PyThon)